tests/spyre_util.py

```diff
@@ -247,14 +248,18 @@ def generate_spyre_vllm_output(
 
 
 # Hugging Face
+# This uses lru_cache to cache the generated text so that we don't have to
+# always load and run the transformers model, nor manage a set of files of
+# expected results.
+@functools.lru_cache
```

Comment:
Should we limit the cache size?

https://docs.python.org/3/library/functools.html#functools.lru_cache

```Python
@functools.lru_cache(maxsize=128)
```

Though, I don't suppose there a reason to be concerned about a growing cache?

Reply:
Yeah I wasn't too concerned because this should be caching relatively small objects